<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page_1">
     <widget class="QPushButton" name="pushButton_authentication">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>320</width>
        <height>240</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../media/sf_shared_VM/320x240.jpg</normaloff>../../../media/sf_shared_VM/320x240.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>320</width>
        <height>240</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="pushButton_Cam1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>105</width>
        <height>105</height>
       </rect>
      </property>
      <property name="text">
       <string>CAM1</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Cam2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>125</y>
        <width>105</width>
        <height>105</height>
       </rect>
      </property>
      <property name="text">
       <string>CAM2</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Cam3">
      <property name="geometry">
       <rect>
        <x>125</x>
        <y>10</y>
        <width>105</width>
        <height>105</height>
       </rect>
      </property>
      <property name="text">
       <string>CAM3</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Cam4">
      <property name="geometry">
       <rect>
        <x>125</x>
        <y>125</y>
        <width>105</width>
        <height>105</height>
       </rect>
      </property>
      <property name="text">
       <string>Cam4</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>70</width>
        <height>220</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>205</y>
        <width>170</width>
        <height>25</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>300</width>
        <height>185</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Tab_Cam1">
       <attribute name="title">
        <string>CAM1</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>85</x>
          <y>0</y>
          <width>150</width>
          <height>155</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QRadioButton" name="radioButton">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>140</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>1080p/30FPS</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>140</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>720p/60FPS</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>140</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>640x480p/60FPS</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>120</y>
           <width>140</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>640x480p/90FPS</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="Tab_Cam2">
       <attribute name="title">
        <string>CAM2</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>85</x>
          <y>0</y>
          <width>150</width>
          <height>155</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QRadioButton" name="radioButton_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>140</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>1080p/30FPS</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_6">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>140</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>720p/60FPS</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_7">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>140</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>640x480p/60FPS</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_8">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>120</y>
           <width>140</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>640x480p/90FPS</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="Tab_Cam3">
       <attribute name="title">
        <string>CAM3</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>85</x>
          <y>0</y>
          <width>150</width>
          <height>155</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QRadioButton" name="radioButton_9">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>140</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>1080p/30FPS</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_10">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>140</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>720p/60FPS</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_11">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>140</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>640x480p/60FPS</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_12">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>120</y>
           <width>140</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>640x480p/90FPS</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="Tab_Cam4">
       <attribute name="title">
        <string>CAM4</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>85</x>
          <y>0</y>
          <width>150</width>
          <height>155</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QRadioButton" name="radioButton_13">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>140</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>1080p/30FPS</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_14">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>140</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>720p/60FPS</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_15">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>140</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>640x480p/60FPS</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_16">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>120</y>
           <width>140</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>640x480p/90FPS</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
